ci/cloudbuild/builds/check-api.sh

```diff
@@ -125,12 +125,16 @@ function check_abi() {
       # by some other number indicating the length of the symbol within the
       # "internal" namespace. See: https://en.wikipedia.org/wiki/Name_mangling
       -skip-internal-symbols "(8internal|_internal|4grpc|6google8protobuf|6google3rpc)\d"
-      # We also ignore the raw gRPC Stub class. The generated gRPC headers that
+      # We ignore the raw gRPC Stub class. The generated gRPC headers that
       # contain these classes are installed alongside our headers. When a new
       # RPC is added to a service, these classes gain a pure virtual method. Our
       # customers do not use these classes directly, so this should not
       # constitute a breaking change.
-      -skip-internal-types "::StubInterface"
+      #
+      # Skip `Options::Data<>` which is a private implementation detail.
+      # Otherwise, we get false positives when a specific type template
+      # parameter is no longer used.
+      -skip-internal-types "(::StubInterface|Options::Data<)"
```

Comment:
Can you fully quality the `Options::Data<` type?

Also: s/less/fewer/ in the tile?

Reply:
> Can you fully quality the Options::Data< type?

No, I think because the inline namespace is encoded. And I don't want to be saying `google::cloud::v2_25::Options::Data<`.

> Also: s/less/fewer/ in the tile?

Done.

Reply:
Maybe `::Options::Data<` to avoid clashes with `EvilOptions::Data<`.

Reply:
Done